docs/docs/guides/integrate/identity-providers/_activate.mdx

```diff
@@ -0,0 +1 @@
+Once you created the IdP, you need to activate it.
```

Comment:
Does it really make sense to generalize 1 line?

Reply:
Better than copying the line 5+ times, IMO :-)